include/deal.II/lac/full_matrix.templates.h

```diff
@@ -1663,7 +1663,9 @@ FullMatrix<number>::print_formatted (
   for (size_type i=0; i<m(); ++i)
     {
       for (size_type j=0; j<n(); ++j)
-        if (std::abs((*this)(i,j)) > threshold)
+        if (std::isnan(std::abs((*this)(i,j))))
```

Comment:
Is the `std::abs` call necessary here? That sholdn't effect whether or not it is a NaN.